<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewElectionDialog</class>
 <widget class="QDialog" name="NewElectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Election</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please select a name (max. 25 characters)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPlainTextEdit" name="descriptionPlaintextEdit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please select a description (max. 255 characters)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="questionLabel">
        <property name="text">
         <string>Questions</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="questionsWidget">
        <item>
         <widget class="QListWidget" name="listQuestions"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="btnQuestionAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuestionRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="votersLabel">
        <property name="text">
         <string>Voters</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QWidget" name="votersWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="votersList"/>
         </item>
         <item>
          <widget class="QWidget" name="votersBtns" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="votersAddBtn">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="votersImportBtn">
              <property name="text">
               <string>Import</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="votersRemoveBtn">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="trusteesLabel">
        <property name="text">
         <string>Trustees</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QWidget" name="trusteesWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="trusteesList"/>
         </item>
         <item>
          <widget class="QWidget" name="trusteesBtns" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="trusteesAddBtn">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="trusteesImportBtn">
              <property name="text">
               <string>Import</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="trusteesRemoveBtn">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ending</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="dtEnding">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>descriptionPlaintextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewElectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewElectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
